main.py

```diff
@@ -101,6 +104,8 @@ def __call__(self, request, response, exception):
       RedirectRoute(r'/predictions', PredictionsHandler, 'predictions', strict_slash=True),
       RedirectRoute(r'/record', RecordHandler, 'record', strict_slash=True),
       RedirectRoute(r'/search', SearchHandler, 'search', strict_slash=True),
+      RedirectRoute(r'/suggest/apiwrite/', SuggestApiWriteController, 'suggets-apiwrite', strict_slash=True),
```

Comment:
Also, `suggets-apiwrite` is spelled wrong here and below, which is missing `-review`